modules/external-load-balancer/main.tf

```diff
@@ -1,54 +1,61 @@
+resource "google_compute_health_check" "application" {
+  name = "${var.prefix}application"
+
+  check_interval_sec = 5
+  description        = "The TFE application health check."
+  https_health_check {
+    port         = var.port_application_tcp
+    request_path = "/_health_check"
+  }
+  timeout_sec = 4
+}
+
 resource "google_compute_backend_service" "application" {
   health_checks = [google_compute_health_check.application.self_link]
   name          = "${var.prefix}application"
 
   backend {
-    group = var.primary_cluster_endpoint_group_self_link
+    group = var.primary_cluster_instance_group_self_link
 
-    balancing_mode = "RATE"
-    description    = "The backend service for the TFE application."
-    max_rate       = 1000
+    balancing_mode        = "RATE"
+    description           = "The TFE primary cluster group."
+    max_rate_per_instance = 333
   }
-  port_name   = "https"
-  protocol    = "HTTPS"
-  timeout_sec = 10
-}
-
-resource "google_compute_health_check" "application" {
-  name = "${var.prefix}application-health-check"
+  backend {
+    group = var.secondary_cluster_instance_group_manager_instance_group
 
-  check_interval_sec = 5
-  tcp_health_check {
-    port = "443"
+    balancing_mode        = "RATE"
+    description           = "The TFE secondary cluster group."
+    max_rate_per_instance = 333
```

Comment:
just wondering what the reason of dropping this to 333 requests per second is (no judgement, just curious)

Reply:
Based on https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/54#discussion_r403130645, I assumed that the secondaries should receive the same rate of traffic. I wonder if we should make these rates variable based on the machine types.